media-video/mpv/mpv-0.11.0-r1.ebuild

```diff
@@ -0,0 +1,278 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_3,3_4} )
+PYTHON_REQ_USE='threads(+)'
+
+WAF_PV='1.8.12'
+
+inherit eutils fdo-mime gnome2-utils pax-utils python-any-r1 waf-utils
+
+DESCRIPTION="Media player based on MPlayer and mplayer2"
+HOMEPAGE="https://mpv.io/"
+SRC_URI="https://waf.io/pub/release/waf-${WAF_PV}"
+
+if [[ ${PV} != *9999* ]]; then
+	SRC_URI+=" https://github.com/mpv-player/mpv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux"
+	DOCS=( RELEASE_NOTES )
+else
+	EGIT_REPO_URI="https://github.com/mpv-player/mpv.git"
+	inherit git-r3
+fi
+DOCS+=( README.md etc/example.conf etc/input.conf )
+
+# See Copyright in source tarball and bug #506946. Waf is BSD, libmpv is ISC.
+LICENSE="GPL-2+ BSD ISC"
+SLOT="0"
+# Here 'opengl' stands for GLX, 'egl' stands for any EGL-based output
+IUSE="+alsa archive bluray cdda +cli doc drm dvb +dvd egl +enca encode +iconv
+jack jpeg lcms +libass libav libcaca libguess libmpv lua luajit openal +opengl
+oss pulseaudio pvr raspberry-pi rubberband samba sdl selinux test v4l vaapi
+vdpau vf-dlopen wayland +X xinerama +xscreensaver xv"
+
+REQUIRED_USE="
+	|| ( cli libmpv )
+	egl? ( || ( X wayland ) )
+	enca? ( iconv )
+	lcms? ( || ( opengl egl ) )
+	libguess? ( iconv )
+	luajit? ( lua )
+	opengl? ( X )
+	pvr? ( v4l )
+	v4l? ( || ( alsa oss ) )
+	vaapi? ( X )
+	vdpau? ( X )
+	wayland? ( egl )
+	xinerama? ( X )
+	xscreensaver? ( X )
+	xv? ( X )
+"
+
+CDEPEND="
```

Comment:
I've actually looked at `autotools-utils` eclass. And `CDEPEND` is used there in an example.
OK, I will use `COMMON_DEPEND` then.